docs/configure/toolbar.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
As mentioned in the triage meeting earlier, you will need to update `toc.js` to add this page under the Configure section of the docs navigation menu. I think putting it in the User Interface sub-section, immediately following `sidebar-and-urls` would make sense.